esphome/components/shelly_dimmer/shelly_dimmer.cpp

```diff
@@ -111,6 +112,30 @@ void ShellyDimmer::setup() {
 
 void ShellyDimmer::update() { this->send_command_(SHELLY_DIMMER_PROTO_CMD_POLL, nullptr, 0); }
 
+void ShellyDimmer::dump_config() {
+  ESP_LOGCONFIG(TAG, "ShellyDimmer:");
+  LOG_PIN("  NRST Pin: ", this->pin_nrst_);
+  LOG_PIN("  BOOT0 Pin: ", this->pin_boot0_);
+
+  ESP_LOGCONFIG(TAG, "  Leading Edge: %b", this->leading_edge_);
```

Comment:
This is in `esphome/core/log.h`
https://github.com/esphome/esphome/blob/dev/esphome/core/log.h#L165-L167
```suggestion
  ESP_LOGCONFIG(TAG, "  Leading Edge: %s", YESNO(this->leading_edge_));
```